examples/io/elekta_average.py

```diff
@@ -0,0 +1,63 @@
+"""
```

Comment:
I brought this up below but the data are just noise, and useful data (e.g., `sample`) are too old.

@jjnurminen can you see if the `bst_phantom_elekta` data are new enough for you to use instead?